src/interface/src/app/treatments/direct-impacts/direct-impacts.component.ts

```diff
@@ -83,10 +84,12 @@ export class DirectImpactsComponent {
 
   breadcrumbs$ = this.treatmentsState.breadcrumbs$;
   treatmentPlan$ = this.treatmentsState.treatmentPlan$;
-
+  activeMetric$ = this.treatmentsState.activeMetric$;
```

Comment:
What is active metric used for? I think is not being used

Reply:
thanks! removed